wgpu-hal/src/gles/queue.rs

```diff
@@ -1249,7 +1235,7 @@ impl super::Queue {
                         )
                     };
                     if let Some(ref blend) = *blend {
-                        unsafe { gl.enable_draw_buffer(index, glow::BLEND) };
+                        unsafe { gl.enable_draw_buffer(glow::BLEND, index) };
```

Comment:
That's a lot of spooky

Reply:
They were flipped all along for the enable call.
For the disable call it might have worked pre https://github.com/grovesNL/glow/pull/10.

Reply:
Yeah, that's why it's spooky :)